solidity/contracts/Oracle.sol

```diff
@@ -14,6 +14,7 @@ contract Oracle is Ownable {
     uint256 amount;
     address addr;
     bytes4 functionId;
+    uint256 cancelExpiration;
```

Comment:
If you use uint64 instead of uint256, the solidity compiler *should* pack addr, functionId, and cancelExpiration into one word and save gas. It's definitely possible, even if the compiler doesn't handle it automatically. Then we'd have until the year 584554051223 before the time value overflowed.

Reply:
Perfect! That makes for a much more acceptable bump in gas costs (<165000 to <167000).